website/blog/2019-07-15-cucumberjs-v5-released.md

```diff
@@ -0,0 +1,108 @@
+---
+title: WebdriverIO V5 now also supports CucumberJS
+author: Wim Selles
+authorURL: http://github.com/wswebcreation
+authorImageURL: https://avatars2.githubusercontent.com/u/11979740?s=460&v=4
+---
+
+We are pleased to announce that we now have support for CucumberJS in WebdriverIO V5!!!! 
+This has been a great challenge for the projectcommitters so we're all very thankful for the time and effort they put into this.
+So normally we would say
+
+    npm install @wdio/cucumber-framework --save-dev
```

Comment:
can we stay consistent with the code examples and have it formatted this way?

<pre>
```sh
npm install @wdio/cucumber-framework --save-dev
```
</pre>

Reply:
Changed it, thanks